apps/ui/modules/reservation.ts

```diff
@@ -44,10 +45,21 @@ export const getDurationOptions = (
     i += intervalSeconds
   ) {
     const hms = secondsToHms(i);
-    const minute = String(hms.m).padEnd(2, "0");
+    const hourString =
+      hms.h !== 0 && hms.h >= 2
+        ? t("common:abbreviations.hour", { count: hms.h })
+        : "";
+    const minuteString = () => {
+      if (hms.h < 2)
+        return t("common:abbreviations.minute", { count: hms.h * 60 + hms.m });
+      if (hms.m !== 0)
+        return t("common:abbreviations.minute", { count: hms.m });
+      return "";
+    };
+    const optionString = `${hourString} ${minuteString()}`;
     timeOptions.push({
-      label: `${hms.h}:${minute}`,
-      value: `${hms.h}:${minute}`,
+      label: optionString,
+      value: `${hms.h}:${String(hms.m).padEnd(2, "0")}`,
```

Comment:
I'd change the `value` to a number (duration in minutes) now. Introduces too many bugs because of the string comparison and `label !== value`

Reply:
That's way outside the scope of this ticket in my opinion. Last time I tried that was when I completely fucked up the calendar. I was trying to convert the duration to minutes when the time format changed.

TBH I don't see what bugs would be introduced simply from `label !== value` here.

Reply:
Not by that, but both are strings and string comparison causes problems. Accidentially picking `label` instead of `value` and TS isn't gonna catch the problem.
You already did it in another place, accidentially changing value comparison even though values didn't change.